api_tests/applications/views/test_application_list.py

```diff
@@ -49,68 +50,65 @@ def setUp(self):
             }
         }
 
-    def test_user1_should_see_only_their_applications(self):
-        res = self.app.get(self.user1_list_url, auth=self.user1.auth)
-        assert_equal(len(res.json['data']),
-                     len(self.user1_apps))
+    def test_user_should_see_only_their_applications(self, app, user, user_app, url):
+        res = app.get(url, auth=user.auth)
+        assert len(res.json['data']) == ApiOAuth2Application.objects.count()
```

Comment:
Ah, my previous comment got lost, so i'll repost: This assertion is inconsistent with the method name. The length of the returned data should be equal to the number of applications owned by `user`, not the count of *all* applications.